### PR TITLE
Added admin editing of patron auth services.

### PR DESCRIPTION
**Pull request by aslagle**

### CODE REVIEW

File: api/millenium_patron.py [hunk 1]
```diff
@@ -63,6 +64,18 @@ class MilleniumPatronAPI(BasicAuthenticationProvider, XMLParser):
     AUTHENTICATION_MODES = [
         PIN_AUTHENTICATION_MODE, FAMILY_NAME_AUTHENTICATION_MODE
     ]
+
+    SETTINGS = [
+        { "key": VERIFY_CERTIFICATE, "label": _("Verify Certificate"), "optional": True },
```
Comment (aslagle) [inline api/millenium_patron.py]: I'd like to remove this setting, or change it to "Don't verify certificate", so it makes more sense if you leave it blank.

Comment (leonardr) [inline api/millenium_patron.py]: We can call it "Ignore certificate problems".

**Comment by leonardr** [comment]
:+1:

File: api/millenium_patron.py (hunk 1 above)
Comment (aslagle) [inline api/millenium_patron.py]: I left it the same internally but changed the labels so it will hopefully be more clear in the admin interface.

Comment (leonardr) [inline api/millenium_patron.py]: Yes, it looks good.

**Pull request merged**